Sources/FaceLiveness/AV/LivenessCaptureSession.swift

```diff
@@ -72,6 +72,15 @@ class LivenessCaptureSession {
         if captureSession?.isRunning == true {
             captureSession?.stopRunning()
         }
+        if let session = captureSession {
+            for input in session.inputs {
+                session.removeInput(input)
+            }
+            for output in session.outputs {
+                session.removeOutput(output)
+            }
+        }
+        captureSession = nil
```

Comment:
small suggestion for avoid optional chaining and then unwrapping the same optional

```swift
guard let session = captureSession else { return }

defer {
    captureSession = nil
}

if session.isRunning { 
    session.stopRunning()
}
for input in session.inputs {
    session.removeInput(input)
}
for output in session.outputs {
    session.removeOutput(output)
}
```

Reply:
I've updated the code based on the suggestion